### PR TITLE
Add multi-robot sim ISAAC13 scenario instructions

### PR DESCRIPTION
**Pull request by trey0**
These instructions are working well for me (at least up to a few panoramas). Sharing for others who would like to test the integrated system and don't have granite table access.

### CODE REVIEW

File: astrobee/survey_manager/survey_planner/readme.md [hunk 1]
```diff
@@ -26,4 +26,142 @@ Don't forget to change the commands to use the target robot's name (bumble is wh
...
+         4. Click "Apply Options". You should see the green checkmark to
+            the right of "Face Forward" turn into a black "X".
+
+   2. Using command-line tools:
+      ```bash
+      # === In terminal 2 ===
+      cat <<EOF >/tmp/configure_cmds.txt
+      # Configure the recording profile
+      for prefix in "" "/honey"; do
+          TOPIC_PREFIX="\${prefix}" "${OPS_REPO}/dock_scripts/hsc/cmd" -c bagger -config "${OPS_REPO}/gds/ControlStationConfig/DataToDisk-ISAAC/SurveyNoDepth.json"
```
Comment (trey0) [inline astrobee/survey_manager/survey_planner/readme.md:102]: This is not working yet pending a PR on the `astrobee_ops` repo that adds `$TOPIC_PREFIX` support.

**Comment by trey0** [comment]
Closes #144

File: astrobee/survey_manager/survey_planner/readme.md (hunk 1 above)
Comment (trey0) [reply astrobee/survey_manager/survey_planner/readme.md:102]: The relevant PR is now merged. (But remember to update `astrobee_ops`!)

**Review by bcoltin — APPROVED** [review]
Thanks!

**Review by marinagmoreira — APPROVED** [review]

**Pull request merged**